tests/testthat/test-ensembleMethods.R

```diff
@@ -161,10 +162,12 @@ test_that("No errors are thrown by a generics for ensembles", {
 
   test_plot_file <- "caretEnsemble_test_plots.png"
   png(test_plot_file)
-  p1 <- autoplot(ens.class)
-  p2 <- autoplot(ens.reg)
-  p3 <- autoplot(ens.class, xvars=c("Petal.Length", "Petal.Width"))
-  p4 <- autoplot(ens.reg, xvars=c("Petal.Length", "Petal.Width"))
+  suppressWarnings({
```

Comment:
What warnings does this give now?

Reply:
In produces many warnings. First of all, it is displayed many times a warning from `caretEnsemble` package:

`Warning message in predict.caretList(object$models, newdata = newdata, na.action = na.action):
"Predicting without new data is not well supported.  Attempting to predict on the training data."`

This is caused because `predict` function is called repeatedly without specifying `newdata` argument.

Also, it is displayed a warning from `lm` method and the following ones repeteadly:

`Warning message:
"The following aesthetics were dropped during statistical transformation: ymin
and ymax.
ℹ This can happen when ggplot fails to infer the correct grouping structure in
  the data.
ℹ Did you forget to specify a `group` aesthetic or to convert a numerical
  variable into a factor?"`

`Warning message in simpleLoess(y, x, w, span, degree = degree, parametric = parametric, :
"pseudoinverse used at -0.005"`

Reply:
got it. so you're quieting a pre-existing warning.  Thx!